platform/osx/display_server_osx.mm

```diff
@@ -2041,6 +2041,12 @@ - (BOOL)canBecomeMainWindow {
 			CGDisplayHideCursor(kCGDirectMainDisplay);
 		}
 		CGAssociateMouseAndMouseCursorPosition(false);
+		WindowData &wd = windows[MAIN_WINDOW_ID];
+		const NSRect contentRect = [wd.window_view frame];
+		NSRect pointInWindowRect = NSMakeRect(contentRect.size.width / 2, contentRect.size.height / 2, 0, 0);
+		NSPoint pointOnScreen = [[wd.window_view window] convertRectToScreen:pointInWindowRect].origin;
+		CGPoint lMouseWarpPos = { pointOnScreen.x, CGDisplayBounds(CGMainDisplayID()).size.height - pointOnScreen.y };
```

Comment:
Is it not possible to stack displays vertically in Marcos?

Reply:
Or am I misinterpreting the reason behind this code

Reply:
It's possible to arrange displays vertically, but it does not matter. This code is needed to convert coordinates to global space (which is relative to the upper-left corner of the main display).

Reply:
I have checked it with vertically aligned displays just to be sure, it works fine.